src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
-        ["st"] = new("Sotho", "Sotho", "st", "sot", TranslationServices.Google),
-        ["sn"] = new("Shona", "chiShona", "sn", "sna", TranslationServices.Google),
+        ["rw"] = new("Kinyarwanda", "Kinyarwanda", "rw", "kin", TranslationServices.Google),
+        ["sa"] = new("Sanskrit", "संस्कृत", "sa", "san", TranslationServices.Google),
+        ["sah"] = new("Yakut", "Саха тыла", "sah", "sah", TranslationServices.Yandex),
         ["sd"] = new("Sindhi", "سنڌي", "sd", "snd", TranslationServices.Google),
         ["si"] = new("Sinhala", "සිංහල", "si", "sin", TranslationServices.Google | TranslationServices.Yandex),
+        ["sjn"] = new("Sindarin", "Eledhrim", "sjn", "sjn", TranslationServices.Yandex), // Not present in Yandex.Cloud
         ["sk"] = new("Slovak", "Slovenčina", "sk", "slk"),
         ["sl"] = new("Slovenian", "Slovenščina", "sl", "slv"),
+        ["sm"] = new("Samoan", "Gagana Sāmoa", "sm", "smo", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
+        ["sn"] = new("Shona", "chiShona", "sn", "sna", TranslationServices.Google),
         ["so"] = new("Somali", "Af Soomaali", "so", "som", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["es"] = new("Spanish", "Español", "es", "spa"),
+        ["sq"] = new("Albanian", "Shqip", "sq", "sqi"),
+        ["sr"] = new("Serbian (Cyrillic)", "Српски", "sr", "srp"),
+        ["sr-Latn"] = new("Serbian (Latin)", "Srpski (latinica)", "sr-Latn", "srp-Latn", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["st"] = new("Sotho", "Sotho", "st", "sot", TranslationServices.Google),
         ["su"] = new("Sundanese", "Basa Sunda", "su", "sun", TranslationServices.Google | TranslationServices.Yandex),
-        ["sw"] = new("Swahili", "Kiswahili", "sw", "swa"),
         ["sv"] = new("Swedish", "Svenska", "sv", "swe"),
-        ["tl"] = new("Tagalog", "Tagalog", "tl", "tgl", TranslationServices.Google | TranslationServices.Yandex),
-        ["tg"] = new("Tajik", "тоҷикӣ", "tg", "tgk", TranslationServices.Google | TranslationServices.Yandex),
+        ["sw"] = new("Swahili", "Kiswahili", "sw", "swa"),
         ["ta"] = new("Tamil", "தமிழ்", "ta", "tam"),
-        ["tt"] = new("Tatar", "Татар", "tt", "tat"),
         ["te"] = new("Telugu", "తెలుగు", "te", "tel"),
+        ["tg"] = new("Tajik", "тоҷикӣ", "tg", "tgk", TranslationServices.Google | TranslationServices.Yandex),
         ["th"] = new("Thai", "ไทย", "th", "tha"),
-        ["tr"] = new("Turkish", "Türkçe", "tr", "tur"),
+        ["ti"] = new("Tigrinya", "ትግር", "ti", "tir", TranslationServices.Bing | TranslationServices.Microsoft),
         ["tk"] = new("Turkmen", "Türkmen Dili", "tk", "tuk", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
+        ["tl"] = new("Tagalog", "Filipino", "tl", "tgl", TranslationServices.Google | TranslationServices.Yandex),
```

Comment:
The native name should be "Tagalog".
```suggestion
        ["tl"] = new("Tagalog", "Tagalog", "tl", "tgl", TranslationServices.Google | TranslationServices.Yandex),
```

Reply:
I'll change this to Pilipino for the native language.
See previous comment.

Reply:
I'm a bit confused because the [page you provided](https://omniglot.com/language/names.htm) for the native names say Tagalog's native name is just Tagalog or Wikang Tagalog. The same can be seen on Wikipedia's page.
![image](https://user-images.githubusercontent.com/39978852/188333721-8ea64add-a752-47c1-9f22-c765c6b64508.png)

Reply:
>   On the other hand, Pilipino, is how the locals from the Philippines refer to themselves, or to their national language.

The above quote comes from the following link:
http://www.hawaii.edu/cps/filipino.html

Reply:
Yeah but that page it's not talking about Tagalog but rather its standarized form, Filipino. What I'm asking is, how is "Tagalog" written in Tagalog? because I'm pretty sure it's not "Pilipino".

Reply:
> How does people refer to "Tagalog" in Tagalog? because I'm pretty sure it's not "Pilipino".

It most certainly is referred to as Pilipino.

As I stated previously, I lived in the Philippines for 4 years (1983-1987).

I considered that my experience might be out dated, but the following links shows it still to be true.
* [An Introduction to Pilipino Phonetics and Pronunciation](https://www.karger.com/Article/Pdf/258574)
* ["Pilipino" is how we, locals from the Philippines, refer to ourselves, or the national language](https://hinative.com/questions/8148444)
* [Pilipino, is how the locals from the Philippines refer to themselves, or to their national language](http://www.hawaii.edu/cps/filipino.html)
* [Filipino and Pilipino are simply different names for Tagalog](https://learningtagalog.com/articles/tagalog_filipino_pilipino_difference.html)